koku/masu/database/sql/reporting_awstags_summary.sql

```diff
@@ -3,7 +3,7 @@ WITH cte_tag_value AS (
         value,
         li.cost_entry_bill_id,
         li.usage_account_id
-    FROM {{schema | sqlsafe}}.reporting_awscostentrylineitem_daily_summary AS li,
+    FROM {{schema | sqlsafe}}.reporting_awscostentrylineitem_daily AS li,
```

Comment:
The daily table here doesn't get populated for Trino mode, and is ultimately going away, so we need to stick with daily_summary here.